apps/web-remix/app/components/pages/interfaces/form/formInterface.reducer.tsx

```
@@ -0,0 +1,129 @@
+            metadata: action.payload.metadata,
+          },
+        },
+      };
+    case 'SET_STATUS':
+      const result = {
+        ...state.outputs,
+        [action.payload.name]: {
+          ...state.outputs[action.payload.name],
+          isCompleted: action.payload.status,
+        },
+      };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)